lxd/db/cluster/auth_groups.go

```diff
@@ -131,9 +129,9 @@ func (g *AuthGroup) ToAPI(ctx context.Context, tx *sql.Tx) (*api.AuthGroup, erro
 // GetIdentitiesByAuthGroupID returns the identities that are members of the group with the given ID.
 func GetIdentitiesByAuthGroupID(ctx context.Context, tx *sql.Tx, groupID int) ([]Identity, error) {
 	stmt := `
-SELECT identities.id, identities.auth_method, identities.type, identities.identifier, identities.name, identities.metadata 
-FROM identities 
-JOIN identities_auth_groups ON identities.id = identities_auth_groups.identity_id 
+SELECT identities.id, identities.auth_method, identities.type, identities.identifier, identities.name, identities.metadata
```

Comment:
query changes shouldnt be included with an unrelated commit message.

Reply:
This was not intentional, I guess it auto removed trailing spaces. Will split into separate commits